custom_components/spotcast/helpers.py

```diff
@@ -8,17 +8,22 @@
 import random
 from functools import partial, wraps
 
+import homeassistant.core as ha_core
 from homeassistant.components.cast.media_player import CastDevice
 from homeassistant.components.spotify.media_player import SpotifyMediaPlayer
 from homeassistant.helpers import entity_platform
+from homeassistant.exceptions import HomeAssistantError
 
 # import for type inference
 import spotipy
 
 _LOGGER = logging.getLogger(__name__)
 
 
-def get_spotify_devices(hass, spotify_user_id):
+def get_spotify_media_player(hass: ha_core.HomeAssistant, spotify_user_id: str) -> SpotifyMediaPlayer:
+    """
+    Get the spotify media player entity from hass.
```

Comment:
I'm trying to clean up code documentation and move it toward the PEP8 format. could you rewrite this comment section using that format?

Reply:
I am not a Python developer so I am not 100% certain on the PEP8 format. I looked it up and I think all I really needed to do was ensure the docstring was all on one line for a single line doc string correct?

Reply:
No problem, I actually just learned that I am not using the PEP8 version of docstring (because there are none... my bad) but the [google convention](https://github.com/NilsJPWerner/autoDocstring/blob/HEAD/docs/google.md). So basically, it's:

````python
"""Returns the sum of two decimal numbers in binary digits.

Args:
    a (int): A decimal integer
    b (int): Another decimal integer

Returns:
    bool: Binary string of the sum of a and b
"""
````

Anyway, no need to rewrite, I'm going to review the code, mostly I needed the merge conflict to be resolved in order to integrate your change.